layer/event_logging.h

```diff
@@ -23,7 +23,7 @@
 
 namespace performancelayers {
 // Set of supported value types for an attribute.
-enum ValueType { kString, kInt64, kVectorInt64, kBool, kDuration };
+enum ValueType { kString, kInt64, kVectorInt64, kBool, kDuration, kTimestamp };
```

Comment:
nit: can you put these in the lexicographic order? This will make it easier to quickly scan looking for some specific name